modules/@apostrophecms/express/index.js

```diff
@@ -441,7 +504,9 @@ module.exports = {
           // See options.csrfExceptions
           if (!req.cookies[self.apos.csrfCookieName] || req.get('X-XSRF-TOKEN') !== req.cookies[self.apos.csrfCookieName] || req.session['XSRF-TOKEN'] !== req.cookies[self.apos.csrfCookieName]) {
             res.statusCode = 403;
-            return res.send('forbidden');
+            return res.send({
```

Comment:
I understand you don't want to give too much information, but a message as `crsf exception` would be useful to find where in the code we get a 403 error (I suspect there are many places, at least in A2 that was the case, and it was hard to track where the error was coming from when your request is rejected).

Reply:
You convinced me. Adding this.